examples/node/cmdline.js

```diff
@@ -0,0 +1,16 @@
+var nomnom = require("nomnom");
+
+exports.parse = function(defaults) {
+    return nomnom
+        .option('host', {
+            abbr: 'i',
+            default: defaults.host,
+            help: 'The Bloomberg API HTTP server hostname'
+        })
+        .option('port', {
+            abbr: 'p',
+            default: defaults.port,
+            help: 'The Bloomberg API HTTP server port'
+        })
+        .parse();
+}
```

Comment:
Ok, so I haven't noticed at first that this executes when the module is loaded. Look at my comment regarding `opt`.